aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/endpoints/AwsEndpointDecorator.kt

```diff
@@ -141,27 +140,13 @@ class AwsEndpointDecorator : ClientCodegenDecorator {
                     rust(
                         """
                         ${section.serviceConfigBuilder}.set_aws_endpoint_resolver(${section.sdkConfig}.endpoint_resolver().clone());
-                        ${section.serviceConfigBuilder}.set_endpoint_url(${section.sdkConfig}.endpoint_url().map(|url|url.to_string()));
                         """,
                     )
                 }
             }
         }
     }
 
-    override fun endpointCustomizations(codegenContext: ClientCodegenContext): List<EndpointCustomization> {
-        return listOf(
-            object : EndpointCustomization {
-                override fun builtInDefaultValue(parameter: Parameter, configRef: String): Writable? {
-                    return when (parameter.builtIn) {
-                        Builtins.SDK_ENDPOINT.builtIn -> writable { rust("$configRef.endpoint_url().map(|url|url.to_string())") }
-                        else -> null
-                    }
-                }
-            },
-        )
-    }
-
```

Comment:
This is no longer needed because it's being added by `PromotedBuiltInsDecorators`, right?